content/blog/java-ocsp-stapling.md

```diff
@@ -1,25 +1,25 @@
 +++
-categories = ["Clients", "Java", "TLS", "OCSP]
+categories = ["Clients", "Java", "TLS", "OCSP"]
 date = "2021-07-06"
-tags = ["java", "ocsp", "stapling", "TLS]
+tags = ["java", "ocsp", "stapling", "TLS"]
 title = "Using OCSP Stapling with the NATS Java Library"
 author = "Scott Fauerbach"
 +++
 
-You already know that the <a href="https://github.com/nats-io/java-nats">NATS Java library</a> can connect to the NATS Server
-with TLS. The NATS Server now supports OCSP Stapling. To that end, I have built an example on how to set up the SSLContext
+# Using OCSP Stapling with the NATS Java Library
+
+You already know that the [NATS Java library](https://github.com/nats-io/java-nats) can connect to the NATS Server
+with TLS. The NATS Server now supports OCSP Stapling. To that end, I have built an example on how to set up the `SSLContext`
 to use with the NATS Java Library. Mileage may vary, and you will need to incorporate your own certificates and key stores,
 but this [OCSP Example](https://github.com/nats-io/java-nats-examples/tree/main/ocsp) will get you most of the way there.
 
-# OCSP Stapling SSLContext
-
-OCSP Stapling has been supported in Java since JDK 8. There is an excellent document from Oracle which describes it in detail.
+## OCSP Stapling SSLContext
 
-[Client-Driven OCSP and OCSP Stapling](https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ocsp.html)
+OCSP Stapling has been supported in Java since JDK 8. This excellent document from Oracle, [Client-Driven OCSP and OCSP Stapling](https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ocsp.html), describes it in detail.
 
 ### Examples
 
-Example of creating the SSLContext can be found in the [OCSP Example Class](src/main/java/io/nats/ocsp/OcspExample.java)
+Example of creating the `SSLContext` can be found in the [OCSP Example Class](src/main/java/io/nats/ocsp/OcspExample.java)
```

Comment:
what is the full URL for the OCSP Example Class?

Reply:
https://github.com/nats-io/java-nats-examples/blob/main/ocsp/src/main/java/io/nats/ocsp/OcspExample.java

Reply:
fixed